assets/js/omniSearch.js

```diff
@@ -420,11 +416,11 @@ $(document).ready(function ($) {
 
     let projectsLastUpdatedElement =
       '<span>Projects: ' +
-      Math.round((Date.now() - projectLastUpdated) / 1000 / 60) +
+      Math.round((Date.now() - projectLastUpdated) / 60000) + // Convert ms to minutes
```

Comment:
I like comments on the line above the code and not i the middle of an expression